package.json

```diff
@@ -38,7 +38,7 @@
     "prettier": "^2.5.1"
   },
   "devDependencies": {
-    "cypress": "^5.6.0",
+    "cypress": "9.5.2",
```

Comment:
^9.5.2 is ok as long as we are in verison 9 of cypress.

Reply:
I thought the docker image for arm64 will lock the Cypress version to exact 9.5.2?

Reply:
Sure. Fixed